plugin/content-script.js

```diff
@@ -323,6 +402,8 @@ function updateHistory(histories) {
 		let ago_last = last_history.findIndex((element) => element === 'ago');
 
 		let change = values_eth[values_eth.length-1]/values_eth[0]*100-100;
+		
+		let [series_sellers_buyers, labels_sellers_buyers] = combine_buyers_sellers(count(buyers), count(sellers), creator);
```

Comment:
The function name `count` seems to imply that it returns an integer, when I believe this function returns an array. Flagging to double check that the types are correct.